trino/dbapi.py

```diff
@@ -78,6 +81,39 @@
 logger = trino.logging.get_logger(__name__)
 
 
+class TimeBoundLRUCache:
+    def __init__(self, capacity: int, ttl_seconds: int):
+        self.capacity = capacity
+        self.ttl_seconds = ttl_seconds
+        self.cache = OrderedDict()
+        self.lock = Lock()
+
+    def get(self, key):
+        with self.lock:
+            if key not in self.cache:
+                return None
+            value, timestamp = self.cache[key]
+            if time() - timestamp > self.ttl_seconds:
```

Comment:
the time isn't enforced until an access is made, deserves to be a comment where the cache is instantiated.

Reply:
Done.